### PR TITLE
Hosting with AWS Amplify

### PR DESCRIPTION
**Pull request by swaminator**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge
- [x] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/guide/deploy.md [hunk 1]
```diff
@@ -13,6 +13,16 @@ The following guides are based on a few shared assumptions:
   }
 }
 ```
+## AWS Amplify
+In this guide we'll walk through how to deploy and host your Gatsby site using the [AWS Amplify Console](https://console.amplify.aws).
```
Comment (ulivz) [inline packages/docs/docs/guide/deploy.md]: Gatsby site？

Comment (swaminator) [reply packages/docs/docs/guide/deploy.md]: oops! My apologies :) I'll fix it right away.

**Comment by swaminator** [comment]
@ulivz my apologies! Definitely not intentional. I love VuePress. In fact during our beta, all our docs were on VuePress. I love the simplicity of the framework!

**Comment by HerrBertling** [comment]
> In fact during our beta, all our docs were on VuePress

Side note, sorry to hijack: If you mean these docs: https://aws-amplify.github.io/ – then please go back to VuePress 😄there are some really strange effects on mobile with the navigation and scroll behaviour in the current docs.

**Comment by swaminator** [comment]
@HerrBertling no not those docs. The Amplify Console adds capabilities for deployments. We had a beta program which used Vuepress :)

**Pull request merged**